spec/facility_locations_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'spec_helper'
+
+RSpec.describe FacilityLocations do
+  before(:each) do
+    @locations = FacilityLocations.new
+    @or_dmv_office_locations = DmvDataService.new.or_dmv_office_locations
+    @ny_dmv_office_locations = DmvDataService.new.ny_dmv_office_locations
+    @mo_dmv_office_locations = DmvDataService.new.mo_dmv_office_locations
+  end
+  
+  describe '#initialize' do
+    it 'can initialize' do
+      expect(@locations).to be_an_instance_of(FacilityLocations)
+    end
+  end
+  
+  describe '#generate_location_data' do
+    it 'can fetch or_dmv_office_locations' do
+      expect(@or_dmv_office_locations).to be_an(Array)
+      expect(@or_dmv_office_locations.size).to eq(59)
+      expect(@or_dmv_office_locations[0][:title]).to eq("Albany DMV Office")
```

Comment:
Or even just expect this to be_a String, since we don't really care if their office name is Albany DMV Office or if they change it to Albany Count DMV Office.